src/index.tsx

```diff
@@ -518,4 +518,19 @@ const clearErrors = (data: MorfiData<any>, field: FormField<unknown>) => {
 
 const useClearErrors = () => useContext(morfiContext).clearErrors;
 
-export const Morfi = { useForm, useField, initialData, notSubmittable, isValidationError, clearErrors, useClearErrors };
+type MorfiSetupOptions = {
+    comparator: <T>(val1: T, val2: T)=> boolean
+};
+const morfiSetupOptions: MorfiSetupOptions = {
+    comparator(val1, val2) {
```

Comment:
please prefer arrow functions and save this one as initial, so it is possible to restore it when calling "configure" with "undefined" comparator